pkg/cmd/step/git/credentials/step_git_credentials.go

```diff
@@ -25,16 +25,20 @@ import (
 )
 
 const (
-	optionOutputFile     = "output"
+	optionOutputFile = "output"
+	// Deprecated
```

Comment:
What would be our plan to eventually remove this? Should we at least create an issue to keep of track and maybe add a 'deprecation-followup' label of similar? One could then on a regular basis review these issues and evaluate of a certain deprecated feature can be removed. 

I think it is pretty much only the SaaS which uses this, so provided you can update in one go, I'd even be tempted to remove directly. WDYT?

Reply:
This command (unfortunately) is embedded elsewhere in the SaaS pipelines so I’m not going to be able to remove it in one go, but should be able to in a few weeks once we’ve migrated over. I’ll create a task to do that

Reply:
Taks to remove #6723